### ISSUE TITLE
[Centralized Forms] [UX] Improve user flow of downloading a form

### ISSUE THREAD

**Issue by aklausmeier** [department-of-veterans-affairs/va.gov-cms#18927]
## Status
**[2025-10-06] [Fran]** Unblocked; stretch for sprint 6, or candidate for sprint 7 for Tim.
**[2025-10-06] [Fran]** Blocked until we know the outcome of the new VA Forms detail template that the Content IA team is working on.

## Description
Prior design and [research](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/initiatives/2021-post-mvp-releases/research/research-findings.md#universal-issues-unrelated-to-accessibility-technology) on the download link within a Find a Form product did not include usability testing after modal was introduced. This modal was a proposed solution to educate a Veteran that a form may or may not be downloaded and may be opened in a new browser instead. 

Fundamentally a user should not have to select "download" twice to be able to download and get the form as good UX. Utilize a mobile first approach. Device vs. desktop may have implications on what may work best. 

Related Figma idea space(not final designs) for [Find a Form](https://www.figma.com/design/k5j6J4x7HG8oiTIS5UHHUY/VA-forms-redesign?node-id=52-1217&t=1TMjZSJ1iMpIm1W7-1)

Related Resources & Support article: [How to download and open a VA.gov PDF form](https://www.va.gov/resources/how-to-download-and-open-a-vagov-pdf-form/)

### User story

**AS A** Veteran
**I WANT** to clearly understand how to download and fill in a form on Find a Forms search results and form detail pages
**SO THAT** I can complete this task with ease and without being asked to download multiple times.

## Acceptance criteria

- [x] Review prior research
- [x] New user flow is designed and informed by prior research and a11y considerations
- [ ] CAIA review
- [ ] Accessibility lead review
- [ ] Design lead review
- [ ] UX research tickets are spun up as next step
- [ ] Consider submitting a tested and successful solution to the VA design system for updated[ Link - Download variation](https://design.va.gov/components/link/#download) guidance

**Comment by jilladams** [comment]
I went ahead and moved this to next refinement, fyi @FranECross @davidmpickett , but not sure where this sits in priority, and it can prob sit at the bottom of that column for a long time, or could park in Backlog. (I forget what the rules are for Drafts, if that's a better place for it to wait.)

**Comment by FranECross** [comment]
@jilladams Thanks! I think refining it and then having it ready in case Jordan/Dave have a slot to work on it sounds great. Thanks!

**Comment by thejordanwood** [comment]
@FranECross I don't believe this will wrap up before my leave next week. Here's my update for whenever this work is able to be picked up again:

- I [created mockups in Figma](https://www.figma.com/design/k5j6J4x7HG8oiTIS5UHHUY/VA-forms-redesign?node-id=52-1216&t=eLqomDImjVi2rHKq-1) of the results and detail pages. I left comments explaining my suggestions on what should change on those pages.
- This will need reviews from our team. If we move forward with adding an alert, we should review the content with CAIA.

**Comment by jilladams** [comment]
@FranECross I'm not sure what we should do with this for end of sprint. Some options: 
* Move to Admin/Workflow for product discussion of next steps, whenever that's appropriate, given Jordan's note above on the Adobe / prior research piece.
* Move to backlog and revisit when it's time to prioritize the follow up work.

**Comment by FranECross** [comment]
@jilladams Thanks for pinging on this! I'll move to the backlog so we can revisit, since this isn't urgent or a quarterly goal at this time.
